gitbooks/features/obsidian-wiki/memory-tree.zh-CN.md

```
@@ -0,0 +1,172 @@
+
+# 记忆树
+
+<figure><img src="../../.gitbook/assets/image.png" alt=""><figcaption><p>记忆树。所有文档的高度压缩视图。</p></figcaption></figure>
+
+记忆树是 OpenHuman 的知识库。它不是一个披着"记忆"外衣的向量数据库，而是一个确定性的、bucket-sealed（桶密封）处理流水线，将你一天中杂乱的数据流——聊天、邮件、文档、集成同步结果——转化为你机器上结构化的、可查询的、带摘要支撑的 Markdown。
+
+## 它做什么
+
+每个连接的源都走同样的流水线：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)